### PR TITLE
qemu_guest_agent:fix qemu-ga uninstall failed issue

### PR DESCRIPTION
**Pull request by xiagao**
gagent_install/uninstall_cmd can't get correct value if qga pkg is installed.

ID:1658889
Signed-off-by: Xiaoling Gao <xiagao@redhat.com>

### CODE REVIEW

**Comment by xiagao** [comment]
(1/1) Host_RHEL.m7.u6.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.qemu_guest_agent.virtio_serial.gagent_install_uninstall.q35: PASS (829.13 s)

File: qemu/tests/qemu_guest_agent.py [hunk 1]
```diff
@@ -1309,13 +1309,14 @@ def setup(self, test, params, env):
         BaseVirtTest.setup(self, test, params, env)
         if self.start_vm == "yes":
             session = self._get_session(params, self.vm)
+            qemu_ga_pkg_path = self.get_qga_pkg_path(test, session, params, self.vm)
+            self.gagent_install_cmd = params.get("gagent_install_cmd") % qemu_ga_pkg_path
+            self.gagent_uninstall_cmd = params.get("gagent_uninstall_cmd") % qemu_ga_pkg_path
```
Comment (sathnaga) [inline qemu/tests/qemu_guest_agent.py]: please explain issue in commit message clearly,
In this case you need a additional check for `Windows` vm as linux vm install/uninstall cmds does not take any additional arguments.

it is better to handle initialization in below part where assignment already happens, may be additional vm login needs to be handled for windows vm.
 

```
class QemuGuestAgentTest(BaseVirtTest):

    def __init__(self, test, params, env):
        BaseVirtTest.__init__(self, test, params, env)

        self._open_session_list = []
        self.gagent = None
        self.vm = None
        self.gagent_install_cmd = params.get("gagent_install_cmd")
        self.gagent_uninstall_cmd = params.get("gagent_uninstall_cmd")

Comment (xiagao) [reply qemu/tests/qemu_guest_agent.py]: sathnaga  could you help review again?

Comment (xiagao) [reply qemu/tests/qemu_guest_agent.py]: sathnaga@linux.vnet.ibm.com could you help review again?

Comment (xiagao) [reply qemu/tests/qemu_guest_agent.py]: @sathnaga could you help review again?

**Comment by xiagao** [comment]
Update, Thanks. @sathnaga 

 (1/1) Host_RHEL.m7.u6.qcow2.virtio_scsi.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.qemu_guest_agent.virtio_serial.gagent_install_uninstall.q35: PASS (968.21 s)

**Pull request merged**